packages/core/src/workers/tasks/argon2i.ts

```diff
@@ -0,0 +1,15 @@
+import crypto from 'node:crypto';
+
+import { argon2i } from 'hash-wasm';
+
+export default async function argon2iEncrypt(password: string): Promise<string> {
+  return argon2i({
+    password,
+    salt: crypto.randomBytes(16),
+    iterations: 8,
+    parallelism: 1,
+    memorySize: 8 * 1024,
+    hashLength: 32,
+    outputType: 'encoded',
+  });
+}
```

Comment:
better to add comments to show this follows the best practice (as the PR description says)